MaxText/configs/base.yml

```diff
@@ -58,6 +58,8 @@ save_config_to_gcs: False
 # Activation dtypes.
 dtype: "bfloat16"
 quantization: "" #defaults to no quantization, i.e. bf16. possible alternative setting is 'int8'
+quantize_kvcache: False
```

Comment:
If quantize_kvcache is set to True, we will use the type from "quantization" setting as the target precision? Is my understanding correct?

Reply:
The two are independent at present. Quantize_kvcache only quantizes to int8 and is independent of the basic quantization config.